usr/share/lib/img_proof/tests/SLES/test_sles_hardened.py

```diff
@@ -62,14 +64,27 @@ def test_sles_hardened(host, get_release_value, is_sles_sap, is_sle_micro):
         )
     )
 
+    scap_report = os.environ.get("SCAP_REPORT", "")
+    if scap_report:
+        if os.path.exists(scap_report) or not scap_report.endswith(".html"):
+            print("Ignoring SCAP_REPORT: {}".format(scap_report), file=sys.stderr)
```

Comment:
Ditto: please use normal print so this message is caught by logging mechanism.

Reply:
Fixed.